File: OrdersService/Messages/ShippingCreatedMessage.cs
﻿using System;

namespace QueuingMessages
{
    public class ShippingCreatedMessage
    {
        public Guid Id { get; set; }
        public DateTime Created { get; set; }
        public Guid OrderId { get; set; }
        public string UserId { get; set; }
    }
}

File: ShippingService/app/messages/shippingCreatedMessage.ts
export class ShippingCreatedMessage {
    public static TypeID: string = 'QueuingMessages.ShippingCreatedMessage:QueuingMessages';
    public TypeID: string = 'QueuingMessages.ShippingCreatedMessage:QueuingMessages';

    constructor(public Id: any, public Created: Date, public OrderId: any, public UserId: any) {
    }
}